corehq/apps/data_pipeline_audit/dbacessors.py

```diff
@@ -154,7 +155,8 @@ def get_sql_form_ids(domain, doc_type, startdate, enddate):
     state = doc_type_to_state[doc_type]
     for db_alias in get_sql_db_aliases_in_use():
         queryset = XFormInstanceSQL.objects.using(db_alias) \
-            .filter(domain=domain, state=state)
+            .filter(domain=domain, state=state) \
+            .exclude(xmlns=DEVICE_LOG_XMLNS)
```

Comment:
is there ever a scenario where we'd actually want to include these?

Reply:
Those aren't supposed to be there https://github.com/dimagi/commcare-hq/pull/21000 so I can't imagine a time where we would want them. they're also ignored in the xform to es pillow, so they shouldn't be there either